src/components/Badge/Badge.st.css

```diff
@@ -36,23 +33,21 @@
     padding: 0 12px;
     line-height: 20px;
     font-size: 12px;
-    border-width: 1px;
-    border-style: solid;
-    background-color: value(badgeBgColor);
-    color: value(badgeTextColor);
-    border-color: value(badgeBorderColor);
 }
 
 .root:priority(default) {
-    background-color: value(defaultBgColor);
-    color: color(value(DefaultTextColor));
+    background-color: value(badgeBgColor);
+    color: value(badgeTextColor);
     border: none;
 }
 
 .root:priority(light) {
     background-color: color(value(LightBgColor));
     color: color(value(LightTextColor));
     border-color: value(lightBorderColor);
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
```

Comment:
I just noticed, that since this priority doesn't have a border, and the others do, we should probably change the `line-height` here.
So if the `.root` `line-height` is `20px`, the `light` priority should probably have `18px` to compensate for the border.
This will make the three priorities have the same height